global-utils/src/main/kotlin/org/yapp/globalutils/DomainService.kt

```
@@ -8,4 +8,4 @@ import org.springframework.stereotype.Component
 @Component
 annotation class DomainService {
 
-}
\ No newline at end of file
+}
```

detekt (1.23.8)

[warning] The class or object DomainService is empty.

(detekt.empty-blocks.EmptyClassBlock)